src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -60,29 +77,40 @@ def ui_term_with_vspace(widget)
       end
 
       def delete_fstab_option!(option)
-        # The options can only be modified using BlkDevice#fstab_options=
-        filesystem.fstab_options = filesystem.fstab_options.reject { |o| o =~ option }
+        # The options can only be modified using MountPoint#mount_options=
+        mount_point.mount_options = mount_point.mount_options.reject { |o| o =~ option }
       end
 
       def add_fstab_options(*options)
-        # The options can only be modified using BlkDevice#fstab_options=
-        # so we can't just append options with << or push
-        new_options = filesystem.fstab_options
-        options.each { |opt| new_options << opt }
-        filesystem.fstab_options = new_options
+        # The options can only be modified using MountPoint#mount_options=
+        mount_point.mount_options = mount_point.mount_options + options
       end
 
       alias_method :add_fstab_option, :add_fstab_options
 
     private
 
+      # Currently editing filesystem
```

Comment:
NP: A bit easy to misunderstand... how about "filesystem currently being edited" ?